src/app.vue

```diff
@@ -97,6 +102,24 @@ if (import.meta.env.MODE !== 'production') {
     meta: [{ name: 'robots', content: 'noindex' }],
   })
 }
+
+const isHistoryBackNavigationDetected = ref(false)
```

Comment:
suggesting shorter naming, which i think it still make sense given the context

```suggestion
const isBackNavigationDetected = ref(false)
```
or
```suggestion
const isNavigateBack = ref(false)
```

Reply:
isBackNavigationDetected sounds better, thanks